### ISSUE TITLE
Subject ctrl-click generates "simple:" query which does not work (regression)

### ISSUE THREAD

**Issue by t7ko** [opto/Expression-Search-NG#119]
This was working some weeks ago, not working any more.

User story: I want to see all emails with given subject.

Steps:
* select an email
* ctrl-click on subject
  * ==> it populates quick-filter query with `simple:the-subject-line-here`

Expected behavior:
* it finds all emails with this subject (or, omg, at least an email I have clicked on)

Actual behavior:
* search results are completely empty, even the currently selected email is not listed.

Note:
* When I manually replace `simple:`  by `s:` -- it works as expected.

plugin version -- latest as of today, (v4.0.23)
thunderbird version 115.11.0 (32-bit)

**Comment by avegaweiss** [comment]
Hi! Based on your note, just a version that does wht your suggests: uses 's' instead of 'simlple'
It seems working, very little testing here!
[expression_search_ng-4.0.34.3-tb.xpi.zip](https://github.com/opto/Expression-Search-NG/files/15441438/expression_search_ng-4.0.34.3-tb.xpi.zip)

**Comment by t7ko** [comment]
Thanks for fixing!  But -- with this fix, now the `Pin filter` feature is now misbehaving.  My actual user scenario was:

* Open "saved search".
* Find an email there.
* "I as a user want to see the whole thread", so:
* CTRL + Right click on subject
  * (Filter is always pinned in my case)
* Click on Inbox which holds all emails.

Expected behavior:

* Filter stays the same -- populated with "search by subject" criteria.  (And shows the whole thread.)

Actual behavior:

* When switching to another folder, no matter what is the current filter -- filter is re-set to a query saved previously.  I cannot reset it actually, it keeps applying the same query whenever I switch between folders/saved searches.

**Comment by avegaweiss** [comment]
Yes, I have the same problem. I'not fully undertanding the logic in the code. I've just skipped some buggy code to enable the plugin working, but something with the pin and/or saved searches is broken.
I'm still trying to figure out....but never workie on a thunder plugin before, so I'm a bit confused.

**Comment by darwin4ever** [comment]
> This was working some weeks ago, not working any more.
> 
> User story: I want to see all emails with given subject.
> 
> Steps:
> 
> * select an email
> * ctrl-click on subject
>   
>   * ==> it populates quick-filter query with `simple:the-subject-line-here`
> 
> Expected behavior:
> 
> * it finds all emails with this subject (or, omg, at least an email I have clicked on)
> 
> Actual behavior:
> 
> * search results are completely empty, even the currently selected email is not listed.
> 
> Note:
> 
> * When I manually replace `simple:`  by `s:` -- it works as expected.
> 

I can confirm above, I have exactly the same problem 
Similar functionality is available for the from-field. There both **from: and f:** are working correctly

plugin version -- latest as of today, (v4.0.34) thunderbird version 115.14.0 (64-bit)
